providers/fpm_pool.rb

```diff
@@ -70,11 +70,16 @@ def register_fpm_service
       fpm_pool_chdir: new_resource.chdir,
       fpm_pool_additional_config: new_resource.additional_config
     )
-    notifies :restart, "service[#{node['php']['fpm_service']}]"
+    # Move this to an independant action
+    # notifies :restart, "service[#{node['php']['fpm_service']}]"
   end
   new_resource.updated_by_last_action(t.updated_by_last_action?)
 end
 
+action :restart do
+  notifies :restart, "service[#{node['php']['fpm_service']}]"
```

Comment:
This can't actually be done this way. Notifies is a property in Chef resources, but can't stand alone in an action.